### ISSUE TITLE
As Administrator, I want to get the list of All SyncUser

### ISSUE THREAD

**Issue by kgiroux** [realm/realm-mobile-platform#89]
Hi,

It is a feature, that concerned http://stackoverflow.com/questions/41514358/is-it-possible-to-get-list-of-syncuser-as-administrator

I would like, when i am connected as Administrator, can get the list of all SyncUser, and create for example a list where i can administrate, and change all permission. 

Here, the purpose of this "issue", is more to provide the API for doing that, not the implementation of the logic for retrieving the list. 

Actually, we have (in java), (I didn't check for the other language), an API `SyncUser.all();`  that retrieved all SyncUser that was connected to the current device. 

Can we add, the following API (It is only a example) : 

`SyncUser.all(ServerConfigurationURL); `

**Comment by sorenvind** [comment]
Hi @kgiroux and thanks for the great suggestion! We're planning on providing an API for looking up users, but can't share a specific timeline at this time. 

In order to be able to implement your use case, I suppose you would like to be able to use all user-management functionality on the client (creating, updating, changing, deleting and listing)?

**Comment by kgiroux** [comment]
Yes it would be great to have the user Management functionality. 

If it is possible, it should be great to allow administrator to do backup of user Realm.

For example, if i want to modify as an administrator the data of one of my user, it should be "cool" to create a copy, modify it, and if it failed, revert. 

But it is a another API i think

**Comment by bigfish24** [comment]
@kgiroux we do offer a generic backup solution: https://realm.io/docs/realm-object-server/#overview

**Comment by bigfish24** [comment]
You can also administer the permissions manually by inspecting the [Admin Realm](https://realm.io/docs/realm-object-server/#admin-realm) which is a Realm used by the server to store the users and permissions. You can access this with an admin user via the [Realm Browser](https://realm.io/docs/realm-object-server/#data-browser) via the `Connect to Object Server` option and open `__admin`.
